packages/app/src/components/catalog/EntityPage.tsx

```diff
@@ -70,9 +70,11 @@ import { EntityTechdocsContent } from '@backstage/plugin-techdocs';
 import { ReportIssue } from '@backstage/plugin-techdocs-module-addons-contrib';
 import { TechDocsAddons } from '@backstage/plugin-techdocs-react';
 import {
+  EntityCatalogInfoQueryCard,
   EntityDqlQueryCard,
   EntityKubernetesDeploymentsCard,
 } from '@dynatrace/backstage-plugin-dql';
+import { EntitySrgValidationsCard } from '@dynatrace/backstage-plugin-dql/src/plugin';
```

Comment:
Why do we need a separate `EntitySrgCalidationsCard`? I assumed to go with the default card and just provide an additional DQL query.

Reply:
The `EntitySrgValidationsCard` is basically the same as the default card with the advantage that the EmptyPage is according to the requirement promoting the SRG.

When the EntitySrgValidationsCard is removed, the user must specify the EmptyPage in the EntityPage.tsx file which I assume is not optimal.

I have 2 possible suggestions:
- Including a `if(queryId === 'dynatrace.srg-validations')` in the `DqlQuery` for rendering the empty state
- Or keeping the EntitySrgValidationsCard

Reply:
Let`s go with the first suggestion and add the if-statement in favor of keeping the EntitySRGValidationCard.